check_mk_web_api/test_WebApi.py

```diff
@@ -527,3 +527,23 @@ def test_logout_site():
     api.add_user('user00', 'User 00', 'p4ssw0rd')
     api.login_site('cmk', 'user00', 'p4ssw0rd')
     api.logout_site('cmk')
+
+@pytest.mark.skip(reason="test_bulk_discovery_all_hosts")
```

Comment:
Why skip this test?

Reply:
It is not possible to start 2 bulk_discoveries at the same time.
Because `test_bulk_discovery_start` runs before `test_bulk_discovery_all_hosts`, the second test would fail.
It is also possible to call `test_bulk_discovery_status` in a loop and to a sleep until the discovery is done, but that would slow down the test.

`test_bulk_discovery_all_hosts` calls `test_bulk_discovery_start` anyway, so just added it for completeness.

Reply:
In this case I'd prefer to loop and wait until the discovery is finished. If we include this function I'd like to have it tested.
Think it shouldn't take too long anyway.

Reply:
Done in b254bd95ed1c721a463e36e90e767047b019a72c

Reply:
Looking good :+1: 
Now we just need to tackle the version issue and we are ready to merge.